kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/MobiComConversationFragment.java

```diff
@@ -1090,9 +1098,8 @@ public void onChannelUpdated() {
 
         //If user has Integrated Zopim, initialize Zendesk Chat SDK
         Contact assigneeContact = appContactService.getContactById(channel.getConversationAssignee());
-        String zendeskChatSdkKey = KmAppSettingPreferences.getInstance().getZendeskSdkKey();
-        if(!TextUtils.isEmpty(zendeskChatSdkKey) && assigneeContact != null && User.RoleType.AGENT.getValue().equals(assigneeContact.getRoleType()) ) {
-            KmZendeskClient.getInstance(getContext()).initializeZendesk(zendeskChatSdkKey, channel.getKey(), appContactService.getContactById(MobiComUserPreference.getInstance(getContext()).getUserId()), channel);
+        if(!TextUtils.isEmpty(zendeskChatSdk) && assigneeContact != null && User.RoleType.AGENT.getValue().equals(assigneeContact.getRoleType()) ) {
```

Comment:
this will work if the assignment is happening to admin/Super admin of the account?

Reply:
Yes, I have tested it with super admin, agents and operators. It is working for any human assignment.